tools/autograd/templates/VariableType.h

```diff
@@ -42,14 +42,16 @@ using at::Quantizer;
 using ConstQuantizerPtr = const c10::intrusive_ptr<Quantizer>&;
 using c10::optional;
 
+// TODO: Once we have C++17, this can be a template function rather than a macro.
+#define UNPACK_TENSOR( t, pos ) \
+  TORCH_CHECK( t.defined(), "Expected a Tensor of type Variable but found an undefined Tensor for argument #", pos, " '", #t, "'" ); \
```

Comment:
use compile-time string literal concatenation -- `" '" #t "'"`

Reply:
if pos is a literal we should stringify it at compile-time too

Reply:
Done.